Fix/escaped passwords

jonatanhanson opened the pull request:
Strip slashes from the password that is sent to AD when logging in user. 
Fixes the bug where some users could not login.
To test it you need to login with a user password that contains special characters like single quotes.

jonatanhanson committed:
Strip slashes from password that will be sent to AD

jonatanhanson committed:
prettier formatting

helsingborgsstad commented:
Task linked: [CU-92y8ug Prio 1: Vissa användare kan inte logga in på intranätet](https://app.clickup.com/t/92y8ug)

jonatanhanson committed:
Escape only characters that is required by Active Directory

Escaping is needed for following characters: , \ # + < > ; " = Leading or trailing spaces

jonatanhanson committed:
Escape leading and trailing spaces from the pw

yo-l1982 reviewed (approved):
Work with my user.
